department-of-veterans-affairs/caseflow
# Remove "Assign to Team" action for BvaDispatchTasks

Resolves #13281

### Description

More palatable error msg: https://github.com/department-of-veterans-affairs/caseflow/issues/13281#issuecomment-579436969

BvaDispatchTask should not offer "Assign to Team" action.

### Acceptance Criteria
- [ ] "Assign to Team" is not in the "Select an action ..."  dropdown list.
- [ ] Follow-on ticket created to reassess what actions should be available by default: #13284 

### Testing Plan
1. Switch to user BVAGWHITE
1. Go to their Queue
1. Select one of the cases where the current task is "BVA Dispatch Task"
1. Click on "Select an action ..." dropdown.
1. Note that "Assign to Team" is not in the dropdown list.

### User Facing Changes

Modified error msg before fix was made: https://github.com/department-of-veterans-affairs/caseflow/issues/13281#issuecomment-579436969 

Before fix:
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/55255674/73302975-2b667680-41db-11ea-8aab-652af3a2ea04.png">

After fix:
TBD
<img width="1136" alt="image" src="https://user-images.githubusercontent.com/55255674/73306173-4b993400-41e1-11ea-9709-670f2c8df83d.png">
